doc/man7/ossl-removed-api.pod

```diff
@@ -88,6 +88,102 @@ This includes consulting the L<ossl-guide-migration(7)> documentation.
 
 =item EVP_CIPHER_meth_get_ctrl (Deprecated in 3.0.0) - consult L<ossl-guide-migration(7)>
 
+=item EVP_PKEY_meth_find (Deprecated in 3.0.0) - consult L<ossl-guide-migration(7)>
+
+=item EVP_PKEY_meth_new (Deprecated in 3.0.0) - consult L<ossl-guide-migration(7)>
+
+=item EVP_PKEY_meth_get0_info (Deprecated in 3.0.0) - consult L<ossl-guide-migration(7)>
+
+=item EVP_PKEY_meth_copy (Deprecated in 3.0.0) - consult L<ossl-guide-migration(7)>
+
+=item EVP_PKEY_meth_free (Deprecated in 3.0.0) - consult L<ossl-guide-migration(7)>
+
+=item EVP_PKEY_meth_add0 (Deprecated in 3.0.0) - consult L<ossl-guide-migration(7)>
+
+=item EVP_PKEY_meth_remove (Deprecated in 3.0.0) - consult L<ossl-guide-migration(7)>
+
+=item EVP_PKEY_meth_get_count (Deprecated in 3.0.0) - consult L<ossl-guide-migration(7)>
+
+=item EVP_PKEY_meth_get0 (Deprecated in 3.0.0) - consult L<ossl-guide-migration(7)>
+
+=item EVP_PKEY_meth_set_init (Deprecated in 3.0.0) - consult L<ossl-guide-migration(7)>
+
+=item EVP_PKEY_meth_set_copy (Deprecated in 3.0.0) - consult L<ossl-guide-migration(7)>
+
+=item EVP_PKEY_meth_set_cleanup (Deprecated in 3.0.0) - consult L<ossl-guide-migration(7)>
+
+=item EVP_PKEY_meth_set_paramgen (Deprecated in 3.0.0) - consult L<ossl-guide-migration(7)>
+
+=item EVP_PKEY_meth_set_keygen (Deprecated in 3.0.0) - consult L<ossl-guide-migration(7)>
+
+=item EVP_PKEY_meth_set_verify (Deprecated in 3.0.0) - consult L<ossl-guide-migration(7)>
```

Comment:
EVP_PKEY_meth_set_sign() missing?

Reply:
Fixed! I also reaslised that I had forgotten to add all these functions to the NAME section. So, I've added fixed that too